### PR TITLE
Uncomment *_DetectionFreq tests

### PR DESCRIPTION
**Pull request by JamesWanglf**
This PR is for #288 .
Since the dynamic linking for Tensorflow library is available on FFmpeg, [lpms/ffmpeg/api_test.go/detectionFreq](https://github.com/livepeer/lpms/blob/e5409705d81e2ea3c009af81439eb4098e6fa128/ffmpeg/api_test.go#L1401) and [lpms/ffmpeg/nvidia_test.go/TestNvidia_DetectionFreq](https://github.com/livepeer/lpms/blob/e5409705d81e2ea3c009af81439eb4098e6fa128/ffmpeg/nvidia_test.go#L804) will be helpful to have tests to confirm that the detection feature is working well with dynamic linking of Tensorflow library.
- I have enabled the commented part from `api_test.go` and `nvidia_test.go` .
- I changed `install_ffmpeg.sh` to download `Tensorflow` library and `tasmodel.pb` from [livepeer-ml](https://github.com/livepeer/livepeer-ml/releases)
  Since CI does not have GPU yet, it will download `Tensorflow C library`(cpu support) for now.
- I updated the latest commit id of FFmpeg.

### CODE REVIEW

**Review by oscar-davids — APPROVED** [review]
LGTM!

File: install_ffmpeg.sh [hunk 1]
```diff
@@ -47,6 +48,7 @@ else
     echo "clang detected, building with GPU support"
 
     EXTRA_FFMPEG_FLAGS="--enable-cuda --enable-cuda-llvm --enable-cuvid --enable-nvenc --enable-decoder=h264_cuvid --enable-filter=scale_cuda,signature_cuda --enable-encoder=h264_nvenc"
+    # LIBTENSORFLOW_SUPPORT="gpu"
```
Comment (cyberj0g) [inline install_ffmpeg.sh:51]: Can we confirm that CPU version of Tensorflow is required when there's no GPU? Official site states something like 'with GPU support' for GPU version, which doesn't mean it won't work in CPU mode, and same is my personal experience. Would be great to check whether we can just use GPU version in both cases.

Comment (JamesWanglf) [reply install_ffmpeg.sh]: I have tested with GPU version of Tensorflow on the device that does not have GPU. Actually, GPU version is working for both cases.

**Comment by cyberj0g** [comment]
LGTM!

**Comment by JamesWanglf** [comment]
I had a test how the dynamic linking of Tensorflow works by using (livepeer_bench)[https://github.com/livepeer/go-livepeer/blob/master/cmd/livepeer_bench/livepeer_bench.go] on my local GPU server.
`./livepeer_bench -in ./data/input.m3u8 -repeat 1000 -nvidia 0 -detectionFreq 1 -transcodingOptions P720p30fps16x9 -outPrefix output`
It gave me the logs like the followings as the result of each repeat.
```
2022-01-21 18:59:22.5657,0,0,10.24,5.139,map[0:0.00995695125311613 1:0.7822408080101013]
2022-01-21 18:59:31.4782,0,1,9.92,3.808,map[0:0.0006493143737316132 1:0.9373460412025452]
2022-01-21 18:59:41.481,0,2,10.24,3.886,map[0:0.00015749999147374183 1:0.964136004447937]
2022-01-21 18:59:51.9458,0,3,10.24,4.106,map[0:0.0003276171628385782 1:0.9671741724014282]
2022-01-21 19:00:01.9061,0,4,10.08,3.826,map[0:0.0006490476080216467 1:0.7747275829315186]
2022-01-21 19:00:12.0381,0,5,10,3.877,map[0:0.0011259631719440222 1:0.9529402852058411]
2022-01-21 19:00:21.8622,0,6,10,3.701,map[0:0.0003577200404834002 1:0.9418689012527466]
2022-01-21 19:00:31.9188,0,7,9.84,3.756,map[0:0.0003207316913176328 1:0.9161058068275452]
2022-01-21 19:00:41.6644,0,8,9.6,3.656,map[0:0.0012649581767618656 1:0.7778319716453552]
2022-01-21 19:00:51.6715,0,9,10.64,4.06,map[0:0.0008925176807679236 1:0.9385285377502441]
2022-01-21 19:01:02.2106,0,10,9.88,3.958,map[0:0.002191700506955385 1:0.7244361042976379]
2022-01-21 19:01:11.5442,0,11,9.32,3.407,map[0:0.0016657084925100207 1:0.20143482089042664]
2022-01-21 19:01:21.2226,0,12,10,3.762,map[0:0.0017923597479239106 1:0.005496361758559942]
*---------------------------------*------------------*
| Concurrent Sessions               | 1                        |
| Total Segs Transcoded            | 13                      |
| Real-Time Segs Transcoded   | 13                      | 
| * Real-Time Segs Ratio *        | 1                        |
| Total Source Duration            | 130s                   |
| Total Transcoding Duration    | 50.943080764s  |
| * Real-Time Duration Ratio *  | 0.3919               |
*---------------------------------*------------------*
```
It was repeated 1000 times. During this test, the GPU's memory usage was always kept at 2.5 Gb. 
As a result of test, I think we can confirm that the dynamic linking is working stably.

**Review by cyberj0g — APPROVED** [review]
LGTM

**Pull request merged**